components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/EventManagementService.java

```diff
@@ -35,5 +35,10 @@
 
     public void unregisterEventSync(String syncId);
 
+    public void updateLatestEventSentTime(String publisherName, int tenantId, long timestamp);
+
+    public long getLatestEventSentTime(String publisherName, int tenantId);
+
+    public long getClusterTimeInMilies();
```

Comment:
Thanks @nirmal070125, this has been corrected by @suhothayan in commit - https://github.com/wso2/carbon-analytics-common/commit/507e6b07e7e4cf7b77d96d42c4f9cd0be0c9edf4